frontend/src/components/DiagnosticSummary/DiversificationMeasureSummary.vue

```diff
@@ -50,38 +50,47 @@
           </div>
         </li>
         <li v-else>
-          <v-icon color="primary" class="mr-2">$close-line</v-icon>
+          <v-icon color="primary" class="mr-2">$question-line</v-icon>
           <div>
             Je n'ai pas renseigné les bases utilisées pour mon menu végétarien
           </div>
         </li>
       </div>
 
-      <li v-if="displayDiversificationPlanSegment && diagnostic.hasDiversificationPlan">
-        <v-icon color="primary" class="mr-2">$check-line</v-icon>
-        <div>
-          J'ai mis en place un plan pluriannuel de diversification des protéines incluant des alternatives à base de
-          protéines végétales
-          <ul role="list" class="mt-2" v-if="appliedDiversificationActions && appliedDiversificationActions.length">
-            <li class="fr-text-xs mb-1" v-for="action in appliedDiversificationActions" :key="action">
-              {{ action }}
-            </li>
-          </ul>
-        </div>
-      </li>
-      <li v-else-if="displayDiversificationPlanSegment">
-        <v-icon color="primary" class="mr-2">$close-line</v-icon>
-        <div>
-          Je n'ai pas mis en place un plan pluriannuel de diversification des protéines incluant des alternatives à base
-          de protéines végétales
-        </div>
+      <li v-if="displayDiversificationPlanSegment">
+        <span v-if="diagnostic.hasDiversificationPlan">
+          <v-icon color="primary" class="mr-1">$check-line</v-icon>
+          <span>
+            J'ai mis en place un plan pluriannuel de diversification des protéines incluant des alternatives à base de
+            protéines végétales
+            <ul role="list" class="mt-2" v-if="appliedDiversificationActions && appliedDiversificationActions.length">
+              <li class="fr-text-xs mb-1" v-for="action in appliedDiversificationActions" :key="action">
+                {{ action }}
+              </li>
+            </ul>
+          </span>
+        </span>
+        <span v-else-if="nullAsFalse || diagnostic.hasDiversificationPlan === false">
+          <v-icon color="primary" class="mr-1">$close-line</v-icon>
+          <span>
+            Je n'ai pas mis en place un plan pluriannuel de diversification des protéines incluant des alternatives à
+            base de protéines végétales
+          </span>
+        </span>
+        <span v-else>
+          <v-icon color="primary" class="mr-1">$question-line</v-icon>
+          <span>
+            Est-ce que vous avez mis en place un plan pluriannuel de diversification des protéines incluant des
```

Comment:
Ça t'irait de mettre ces labels avec une inversion de verbe ? C'est un peu plus court et facile à lire. Car la ligne ne commencerait pas toujours avec les mêmes chars.

J'ai ajouté quelques suggestions supplémentaires mais je ne l'ai pas fait pour tout car je ne veux pas trop remplir cette PR avec la même suggestion.
```suggestion
            Avez-vous mis en place un plan pluriannuel de diversification des protéines incluant des
```